app/frontend/src/features/Home.tsx

```diff
@@ -1,22 +1,78 @@
+import { Link, Typography } from "@material-ui/core";
 import Button from "components/Button";
+import Divider from "components/Divider";
 import TextBody from "components/TextBody";
+import {
+  BUGS,
+  BUGS_DESCRIPTION_1,
+  BUGS_DESCRIPTION_2,
+  BUGS_DESCRIPTION_3,
+  COMMUNITY_FORUM,
+  COMMUNITY_FORUM_LINK,
+  CREATE_GUIDE,
+  CREATE_PLACE,
+  FEATURES,
+  FEATURES_DESCRIPTION,
+  HELP,
+  HELP_DESCRIPTION_1,
+  HELP_DESCRIPTION_2,
+  SIGN_UP,
+  SIGN_UP_LINK,
+  WELCOME,
+  WELCOME_DESCRIPTION_1,
+  WELCOME_DESCRIPTION_2,
+} from "features/constants";
 import useCurrentUser from "features/userQueries/useCurrentUser";
 import React from "react";
-import { Link } from "react-router-dom";
+import { Link as RouterLink } from "react-router-dom";
 import { newGuideRoute, newPlaceRoute } from "routes";
 
 export default function Home() {
   const name = useCurrentUser().data?.name.split(" ")[0];
+  const isPostBetaEnabled = process.env.REACT_APP_IS_POST_BETA_ENABLED;
 
-  return (
+  return isPostBetaEnabled ? (
     <>
       {name ? <TextBody>Hello, {name}.</TextBody> : null}
-      <Button component={Link} to={newPlaceRoute}>
-        Create a new place
+      <Button component={RouterLink} to={newPlaceRoute}>
+        {CREATE_PLACE}
       </Button>
-      <Button component={Link} to={newGuideRoute}>
-        Create a new guide
+      <Button component={RouterLink} to={newGuideRoute}>
+        {CREATE_GUIDE}
       </Button>
```

Comment:
I wonder if there's much value keeping this around and if it's better/easier to just delete it, as I doubt we'll want to keep any of this when we build out the dashboard properly. What do you think?

Reply:
Yeah I was thinking that too, but I wasn't sure whether we would actually need it. Guess we can just make it when / if we do!